File: Backend/Test/Small_Files/sample.java
/*
 * This is a sample Java file for testing purposes.
 * It contains a simple class with a main method.
 */

public class Sample {
    public static void main(String[] args) {
        String[] names = {"Alice", "Bob", "Charlie", "Diana", "Eve"};
        for (String name : names) {
            System.out.println(greet(name));
        }
    }

    public static String greet(String name) {
        return "Hello, " + name + "!";
    }
}

File: Backend/Test/Small_Files/sample.py
"""
This is a sample Python file for testing purposes.
It contains a simple function and a loop to generate output.
"""

def greet(name: str) -> str:
    """
    Returns a greeting message for the given name.
    """
    return f"Hello, {name}!"

if __name__ == "__main__":
    names = ["Alice", "Bob", "Charlie", "Diana", "Eve"]
    for name in names:
        print(greet(name))